catwalk/steps.py

```diff
@@ -199,7 +199,13 @@ def run(
             ]
         dataset_dict = DatasetDict(splits=splits, metadata={})
 
-        trainable_model = model.trainable_copy()
+        trainable_model_kwargs: Dict[str, Any] = {}
+        for task in tasks_in_a_special_variable_because_mypy_is_insane:
+            if isinstance(task, WithAnswerOptionsMixin):
+                trainable_model_kwargs["num_classification_labels"] = max(
+                    trainable_model_kwargs.get("num_classification_labels", 0),
```

Comment:
When would we want the default number of class labels to be 0?

Reply:
It's `0` so that the other option in the `max()` function applies, which is `len(task.answer_options)`.